pkg/scyllaclient/client_rclone.go

```diff
@@ -460,22 +463,42 @@ func (c *Client) RcloneListDirIter(ctx context.Context, host, remotePath string,
 		}
 	}
 
+	more := make(chan bool)
```

Comment:
The boolean value is actually being used. If `dec.More()` is true it passes true to the channel so that the timer restarts. If it is false it passes false through the channel so that the loop below can finish execution and return from the function.

Reply:
I see, I guess it could be refactored to a done channel that you would close as there is nothing more to do.